lib/slack/real_time/client.rb

```diff
@@ -125,6 +127,7 @@ def socket_class
 
       def send_json(data)
         fail ClientNotStartedError unless started?
+        logger.debug("#{self.class.name}#send_data") { data }
```

Comment:
:+1: 

Generally I wonder about logging data here, it's a binary blob most of the time. Thoughts?